app/views/initiatives/_initiative.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="card">
+  <h3 class="init-name"><%= initiative.title %></h3>
+  <p class="init-description no-link"><%= simple_format(initiative.description) %></p>
+  <div>
+    <div class="init-footer w-row">
+      <div class="w-col w-col-2 padding-zero"><%= image_tag "icon-#{initiative.completion ? 'completion': 'calendar'}.svg", class: "icon-target" %></div>
+      <div class="card-date w-col w-col-10">
+        <%= initiative.target_date.to_s(:long_ordinal) unless initiative.target_date.nil? %>
+      </div>
+      <div class="w-col w-col-2 padding-zero">
+        <% if !current_user.team_members.pluck(:id).include?(params[:user_id].to_i) %>
+          <%= link_to initiative_path(initiative), method: :delete, data: { confirm: "Are you sure you would like to delete Initative: #{initiative.title}" }, :class => "no-link", :id => "delete-initiative" do %>
+            <%= image_tag "icon-delete.svg" %>
+          <% end %>
+        <% end %>
+      </div>
+    </div>
+    <hr>
+    <div>
+      <%= render initiative.notes.order('created_at DESC') %>
```

Comment:
How could you change this so the the `order` is not in the view?

Reply:
Resolved!